corehq/apps/app_manager/templates/app_manager/partials/modules/case_list.html

```diff
@@ -143,6 +143,79 @@ <h4 class="panel-title panel-title-nolink">
+                <div class="btn-group" role="group">
+                  <button type="button" class="btn btn-xs btn-default"
+                          data-bind="click: function () { $parent.adjustTileGridArea($index(), 0, -1, 0, 0); }, enable: tileColumnStart() > 1">
+                    <i class="fa fa-arrow-left"></i>
+                  </button>
+                  <button type="button" class="btn btn-xs btn-default"
+                          data-bind="click: function () { $parent.adjustTileGridArea($index(), -1, 0, 0, 0); }, enable: tileRowStart() > 1">
+                    <i class="fa fa-arrow-up"></i>
+                  </button>
+                  <button type="button" class="btn btn-xs btn-default"
+                          data-bind="click: function () { $parent.adjustTileGridArea($index(), 1, 0, 0, 0); }, enable: tileRowEnd() < tileRowMax()">
+                    <i class="fa fa-arrow-down"></i>
+                  </button>
+                  <button type="button" class="btn btn-xs btn-default"
+                          data-bind="click: function () { $parent.adjustTileGridArea($index(), 0, 1, 0, 0); }, enable: tileColumnEnd() < tileColumnMax()">
+                    <i class="fa fa-arrow-right"></i>
+                  </button>
+                </div>
+                <div class="btn-group" role="group" style="float: left; margin: 0 2px;">
+                  <button type="button" class="btn btn-xs btn-default"
+                          data-bind="click: function () { $parent.adjustTileGridArea($index(), 0, 0, -1, 0);}, enable: tileWidth() > 1">
+                    <i class="fa fa-minus"></i>
+                  </button>
+                  <button type="button" class="btn btn-xs" disabled>
+                    <i class="fa fa-arrows-h"></i>
+                  </button>
+                  <button type="button" class="btn btn-xs btn-default"
+                          data-bind="click: function () { $parent.adjustTileGridArea($index(), 0, 0, 1, 0); }, enable: tileColumnEnd() < tileColumnMax()">
+                    <i class="fa fa-plus"></i>
+                  </button>
+                </div>
+                <div class="btn-group" role="group" style="float: right; margin: 0 2px;">
+                  <button type="button" class="btn btn-xs btn-default"
+                          data-bind="click: function () { $parent.adjustTileGridArea($index(), 0, 0, 0, -1); }, enable: tileHeight() > 1">
+                    <i class="fa fa-minus"></i>
+                  </button>
+                  <button type="button" class="btn btn-xs" disabled>
+                    <i class="fa fa-arrows-v"></i>
+                  </button>
+                  <button type="button" class="btn btn-xs btn-default"
+                          data-bind="click: function () { $parent.adjustTileGridArea($index(), 0, 0, 0, 1); }, enable: tileRowEnd() < tileRowMax()">
+                    <i class="fa fa-plus"></i>
+                  </button>
```

Comment:
I don't see these buttons in the screen shot.

Reply:
@snopoke  They only show up when you hover over them. I did not find them to bad. Drag and drop might be more intuitive though. 
<img width="839" alt="image" src="https://github.com/dimagi/commcare-hq/assets/1946138/7474be0c-134d-432b-8739-185ef2a2ae8b">